apps/mark/backend/src/app.ts

```diff
@@ -1,18 +1,106 @@
-import * as grout from '@votingworks/grout';
 import express, { Application } from 'express';
+import { z } from 'zod';
+import { InsertedSmartCardAuthApi } from '@votingworks/auth';
+import { err, Result } from '@votingworks/basics';
+import * as grout from '@votingworks/grout';
+import { BallotStyleId, Optional, PrecinctId } from '@votingworks/types';
+import { ScannerReportData, ScannerReportDataSchema } from '@votingworks/utils';
+
 import { getMachineConfig } from './machine_config';
 
-function buildApi() {
+function buildApi(auth: InsertedSmartCardAuthApi) {
   return grout.createApi({
     getMachineConfig,
+
+    // TODO: Once election definition has been moved to the backend, no longer require the frontend
+    // to provide election hash to this and other methods that use the auth lib
```

Comment:
I contemplated moving election definition to the backend but again wanted to limit the scope of this PR

Reply:
Loving that grout makes this a low cost decision!

Reply:
Yes very true! :tada: